package.json

```diff
@@ -44,7 +44,7 @@
     "pandoc-doc-install": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/install.rst wiki/Install.md",
     "pandoc-doc-exchanges": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/exchanges.rst wiki/Exchange-Markets.md",
     "pandoc-doc-exchanges-by-country": "pandoc --wrap=preserve --columns=10000 --from=markdown --to=rst --output=doc/exchanges-by-country.rst wiki/Exchange-Markets-By-Country.md",
-    "postinstall": "node build/postinstall"
+    "postinstall": "node -e 'if (process.cwd ().split (\"/\").pop () !== \"ccxt\") { process.chdir (\"./ccxt\") } require (\"./build/postinstall\") ()'"
```

Comment:
That's too _hacky_, it could be any repo dirname actually, system-specific...

Reply:
It breaks the appveyor for some reason... I'm thinking on a more stable solution in the meantime.

Reply:
It breaks appvoyer because windows is not unix and single quotes mean multiple arguments. I just need a quoting method that works for windows and bash...

Reply:
if we change 'ccxt' to `node_modules` it should work. Basically the reason it is currently broken for `npm install ccxt` is that postinstall is invoked from `node_modules` and `ccxt` when `npm install` is used. Need to change the directory to run the postinstall script... For now you can remove the postinstall message and we can figure out how to fix it later...

Reply:
https://shapeshed.com/writing-cross-platform-node/#scripts-in-package-json

Reply:
that should work, try adding node/postinstall to `package.json` and then update and do npm install ccxt